File: C++/electricaloutlets.cpp
#include <iostream>
using namespace std;

int main() {
  int cases;
  cin >> cases;

  for (int i = 0; i < cases; i++) {
    int strips;
    cin >> strips;

    int sum = 0;
    for (int i = 0; i < strips; i++) {
      int outlets;
      cin >> outlets;

      sum += outlets;
    }

    cout << (sum - (strips - 1)) << endl;
  }
}

File: Java/electricaloutlets.java
import java.io.*;
import java.util.*;

public class electricaloutlets {
  
  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    int cases = Integer.parseInt(br.readLine());

    for (int i = 0; i < cases; i++) {
      StringTokenizer st = new StringTokenizer(br.readLine());
      int strips = Integer.parseInt(st.nextToken());

      int sum = 0;
      while (st.hasMoreTokens()) {
        int outlets = Integer.parseInt(st.nextToken());
        sum += outlets;
      }

      System.out.println(sum - (strips - 1));
    }
  }
}
